org.dupot.beatandmatchtopass.yml

```diff
@@ -0,0 +1,24 @@
+id: org.dupot.beatandmatchtopass
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: beatmatchtopass.x86_64
+finish-args:
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
+  - --device=dri
+  - --persist=.local/share/godot/app_userdata/BeatMatchtoPass
+
+modules:
+  - name: BeatMatchtoPass
+    buildsystem: simple
+    build-commands:
+      - install -Dm555 export/linux/beatmatchtopass.x86_64 $FLATPAK_DEST/bin/beatmatchtopass.x86_64
+      - install -Dm644 export/linux/flatpack/512x512.png $FLATPAK_DEST/share/icons/hicolor/512x512/apps/org.dupot.beatmatchtopass.png
+      - install -Dm644 export/linux/flatpack/org.dupot.beatmatchtopass.desktop $FLATPAK_DEST/share/applications/org.dupot.beatmatchtopass.desktop
+      - install -Dm644 export/linux/flatpack/org.dupot.beatmatchtopass.appdata.xml $FLATPAK_DEST/share/metainfo/org.dupot.beatmatchtopass.appdata.xml
```

Comment:
this doesn't match the app-id in the manifest.

Reply:
i fix it in lasts commit